### ISSUE TITLE
Bookmark for page or query

### ISSUE THREAD

**Issue by Senju-vilis** [hydrusnetwork/hydrus#648]
Ability to add a bookmark to a file in a specific page / query. 

Lets say I browse a page of 1000 files and get to item#213 and then switch pages or restart Hydrus,
When I come back to a page, it is probably reset at the beginning.

Bookmarks would enable quickly jumping back to where you left off.

A bookmark should be accessible even after closing the page its in - 
it would simply reopen that page and jump to the marked file.

**Comment by TheElo** [comment]
As a workaround you could use "Remove Image" from the tag by hiding the images you went thru by selecting the image you are one and the first image, hitting "CTRL+R" to remove them from sight.

Next time you open the tab even after a restart, the image you went thru won´t appear again (doesn´t work for watchers btw)

**Comment by Senju-vilis** [comment]
"CTRL+R" will only remove them from sight in a page but not from the tag itself, right?

**Comment by imtbl** [comment]
> "CTRL+R" will only remove them from sight in a page but not from the tag itself, right?

Yes, this just removes the files from the current page (and only until you run a new search on that page); it doesn't modify the files' tags (or any other data) in any way.

**Comment by imtbl** [comment]
Somewhat related to #618.
